src/features/projectsV2/ProjectsMap/InterventionDropDown/InterventionList.module.scss

```diff
@@ -75,6 +82,15 @@ $layoutPaddingSide: 20px;
     width: 100%;
   }
 }
+
+.interventionListOptionsAbove {
+  top: calc($layoutPaddingTop + 55px);
+}
+
+.interventionListOptionsBelow {
+  top: calc($layoutPaddingTop + 116px);
+}
+
```

Comment:
These calculations need minor adjustment as well, in order to match the top of the sites dropdown and intervention type dropdown (standalone).